Packs/GitHub/Integrations/GitHubEventCollector/GitHubEventCollector.py

```diff
@@ -117,7 +116,9 @@ def main():
 
             if command == 'fetch-events':
                 send_events_to_xsiam(events, vendor=VENDOR, product=PRODUCT)
-                demisto.setLastRun(GithubGetEvents.get_last_run(events))
+                if events:
```

Comment:
please add a unit-test

Reply:
There are no unit tests for this integration at all. We need to have another task to write unit tests for this collector.

Reply:
added unit test for last run